documentation/service_admin/proc-deploying-service-broker.adoc

```diff
@@ -0,0 +1,119 @@
+
+*NOTE*: To install {ProductName} on OpenShift, you must have cluster-admin access to set up the required roles for delegating authentication to the Kubernetes master; otherwise, you are restricted to a single address space. For more information about how to deploy without cluster-admin access, which will restrict {ProductName} to a single address space, see xref:deploying-single-address-space[].
+
+.Prerequisite
+
+* The service broker requires the standard authentication service to be deployed.
+
+.Procedure
+
+. Create a service account for the {ProductName} service broker:
++
+[options="nowrap",subs="attributes"]
+----
+{cmdcli} create sa enmasse-admin
+----
+
+ifeval::["{cmdcli}" == "oc"]
+. Create cluster-wide roles used by the `enmasse-admin` service account:
++
+[options="nowrap"]
+----
+oc login -u system:admin
+oc create -f ./resources/cluster-roles/service-broker.yaml
+----
+
+. Grant privileges to the service account:
++
+[options="nowrap"]
+----
+oc login -u system:admin
+oc policy add-role-to-user admin system:serviceaccount:enmasse:enmasse-admin
+oc adm policy add-cluster-role-to-user enmasse.io:service-broker system:serviceaccount:enmasse:enmasse-admin
+oc adm policy add-cluster-role-to-user system:auth-delegator system:serviceaccount:enmasse:enmasse-admin
+----
++
+*Note*: You can log in again as the regular user after this step.
+endif::[]
+
+. Create a certificate to use for the service broker. For testing purposes, you can create a self-signed certificate:
++
+[options="nowrap"]
+----
+mkdir -p service-broker-cert/
+openssl req -new -x509 -batch -nodes -days 11000 -subj "/O=io.enmasse/CN=service-broker.enmasse.svc.cluster.local" -out service-broker-cert/tls.crt -keyout service-broker-cert/tls.key
+----
+
+. Create a secret containing the service broker certificate:
++
+[options="nowrap",subs="attributes"]
+----
+{cmdcli} create secret tls service-broker-cert --cert=service-broker-cert/tls.crt --key=service-broker-cert/tls.key
+----
+
+. Create a secret containing the service broker secret configuration:
++
+[options="nowrap",subs="attributes"]
+----
+{cmdcli} create secret tls service-broker-secret --from-literal=keycloak.username=admin --from-literal=keycloak.password=admin --from-literal=keycloakCa.crt=`{cmdcli} extract secret/standard-authservice-cert --keys=tls.crt --to=-`
+----
+
+. Deploy the service broker:
++
+[options="nowrap",subs="attributes"]
+----
+{cmdcli} create -f ./resources/service-broker/deployment.yaml
+{cmdcli} create -f ./resources/service-broker/service.yaml
+----
+
+. To ensure the service broker behaves correctly in redirects, you must specify the `keycloakUrl` setting. If you are running a local cluster without a public DNS, use the internal service IP address for the host name; otherwise, use the host name of the external route. To obtain the service IP address, use this command:
```

Comment:
"in redirects" is awkward.  "To ensure the service broker redirects correctly," might be better.